staging/src/k8s.io/apimachinery/pkg/types/uid.go

```diff
@@ -20,3 +20,9 @@ package types
 // don't ONLY use UUIDs, this is an alias to string.  Being a type captures
 // intent and helps make sure that UIDs and names do not get conflated.
 type UID string
+
+// A pod UID which has been translated/resolved to the representation known to kubelets.
+type ResolvedPodUID UID
```

Comment:
@smarterclayton Thanks, I've now done as you recommended. (They were placed there partly because they trivially use it as underlying type, and partly because the changes weren't contained to the kubelet package originally and might not be again after a envisioned second development iteration, but for this PR in isolation it belongs in the kubelet package.)